pennylane/transforms/core/transform_program.py

```diff
@@ -296,15 +307,20 @@ def __call__(self, tapes: Tuple[QuantumTape]) -> Tuple[ResultBatch, BatchPostPro
             batch_postprocessing = partial(_batch_postprocessing, individual_fns=fns, slices=slices)
             batch_postprocessing.__doc__ = _batch_postprocessing.__doc__
 
-            processing_fns_stack.append(batch_postprocessing)
-
             # set input tapes for next iteration.
             tapes = execution_tapes
 
-        postprocessing_fn = partial(
-            _apply_postprocessing_stack,
-            postprocessing_stack=processing_fns_stack,
-        )
+            if is_informative:
+                informative_fn = batch_postprocessing
+                continue
+
+            processing_fns_stack.append(batch_postprocessing)
+
+        def postprocessing_fn(res):
+            if informative_fn:
```

Comment:
I don't think the changes are necessary here. The processing function from the informative transform should be the same. Res can results or tape and I don't think it should be an issue